docs/source/yelpsoa_configs.rst

```diff
@@ -1069,6 +1069,33 @@ internally at Yelp.
      delay. Note that due to Envoy's current definition of *percentage* as an
      integer, this cannot be specified as a floating-point number.
 
+Error Alerting
+``````````````
+
+These keys provide optional overrides for the default alerting behaviour.
+
+ * ``monitoring``: Override default alerting behaviour. For example: ::
+
+      main:
+        monitoring:
+          team: frontend
+          slack_channel: "notifications"
+          jira_project: "FRONTEND"
+          ticket: false
+          page: false
+          page_nonprod: true
+          error_threshold_ration: 0.02
+          minimum_error_rps: 10
+    - ``team``: Override the default team for alerting.
+    - ``slack_channel``: Override the default slack channel for alerting.
+    - ``jira_project``: Override the default JIRA project for alerting.
```

Comment:
should this match monitoring.yaml and be `project`?

Reply:
updated